### DISCUSSION TITLE
QField Windows : mémorisation du projet inopportune

### DISCUSSION THREAD

**Discussion by geguill** [opengisch/QField#4865] [Q&A]
Bonjour,
Soit un projet essai1.gqz (et ses fichiers) ouvert sous QField, consulté et fermé.

Le projet est corrigé sous QGIS (ex paramétrage d'accrochage corrigé). Il est de nouveau ouvert sous QField. Et là, QField ne tient pas compte des modifications ! ?

Si je change le nom du projet en essai2.qgz, pas de problème.
Par contre, si je modifie essai2 avec QGIS, le problème est de nouveau présent sous QField.

J'ai remarqué que QField génère un fichier essai1.qgz.jpg comme une "photo" du dernier canevas sous QField avant fermeture. Je ne pense pas que ce dernier soit responsable de cette "résilience".
J'ai cherché dans C:\Users\xxxx\AppData\Local\Temp (ou dans C:\Users\xxxx\AppData\Local\OPENGIS.ch\QField) un responsable de ce fait ; je n'ai pas trouvé.

Que faut-il faire ?

D'avance merci.
Gérard

**Comment by nirvn** [comment]
@geguill , une fois qu un projet a été ouvert dans QField, l app va mémoriser quelques settings (visibilité des couches, la visibilité des labels, transparence des couches, etc.). 

Que changez vous exactement qui n est pas pris en compte par QField?

**Reply by geguill** [reply]
exemple : les paramétrages d'accrochage d'une couche via dans QGIS le menu  _Projet > Oprtons d'accrochage_
![Capture d'écran 2023-12-18 162144](https://github.com/opengisch/QField/assets/98580457/f3fbcf0c-6edc-4978-8e32-e75e5608a079)
d'accrochage_

**Reply by nirvn** [reply]
Ok, c est définitivement un problème. 

QField utilise le chemin et nom du fichier projet pour sauver et restaurer les settings, c'est pour ça que changer le nom "marche".

**Reply by geguill** [reply]
Cela me rassure un peu !
Mais y a-t-il moyen de jouer sur les settings ?

**Reply by nirvn** [reply]
Pour l'instant la seule solution serait sur Android d effacer le storage afin d "oublier" les settings de projets :

![Screenshot_20231218-223413](https://github.com/opengisch/QField/assets/1728657/5ffdccf8-db01-4a88-8ea4-7a179cc48715)

Vous pouvez aussi renommé votre fichier .qgs/.qgz mais ce n' est pas pratique pour les projets dans le clous.

**Reply by geguill** [reply]
OK pour Androïd. Merci pour l'info.
Pour Windows, n'y aurait-il pas une piste du coté de C:\Users\xxxx\AppData\Local\OPENGIS.ch\QField\cache ?
